media-libs/jbigkit/patches/jbigkit-2.1.patchset

```diff
@@ -0,0 +1,80 @@
+From 267d0811689d2115b42dce345c00be5cd2ccf86d Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 7 Apr 2018 17:31:24 +0200
+Subject: remove -Wno-unused-result, add debug support
+
+
+diff --git a/Makefile b/Makefile
+index 9f7e3ba..645f1c5 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,8 +4,8 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
+-# CFLAGS = -O -g -W -Wall -Wno-unused-result -ansi -pedantic # -DDEBUG
++# CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O -g -W -ansi -pedantic # -DDEBUG
+ 
+ export CC CFLAGS
+ 
+-- 
+2.16.2
+
+
+From 6b76dac0aaa996bdb546ccc51f113ec8bcc719d6 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 7 Apr 2018 17:33:35 +0200
+Subject: add shared objects
+
+
+diff --git a/libjbig/Makefile_so b/libjbig/Makefile_so
+new file mode 100644
+index 0000000..8083541
+--- /dev/null
++++ b/libjbig/Makefile_so
+@@ -0,0 +1,40 @@
++# This Makefile builds a shared version of the library, 
++# libjbig.so.2.1, with soname libjbig.so.2,
++# Please see the README file for some important info 
++# about building the library like this.
++# ------------------------------------------------------------------
++# Please read the WARNING, DISCLAIMER and PATENTS sections in the 
++# README file.
++#
++# This program is released under the terms of the license contained
++# in the file LICENSE.
++# ------------------------------------------------------------------
++
++SHELL=/bin/sh
++CC=gcc
++CFLAGS=-fPIC -g -c -Wall
++
++OBJS=  jbig.o  \
++      jbig_ar.o    \
++      jbig85.o
++
++all: $(OBJS)
++	$(CC) -shared -Wl,-soname,libjbig.so -o libjbig.so.2.1 $(OBJS)
```

Comment:
Hmm, I forgot to mention something: if you declare a `compat >= 2` then I think you should use `-Wl,-soname,libjbig.so.2` instead of `-Wl,-soname,libjbig.so`.
And maybe you would not even need to create one of the 2 symlinks: `libjbig.so ->libjbig.so.2.1` in `$libDir`. Having only `libjbig.so.2 ->libjbig.so.2.1` will probably be OK if I'm not mistaken. In `$developLibDir`, however, I guess you need both symlinks.

cc: @alaviss, as he is an expert.

Reply:
Updated, I'll wait for the reviews :) thanks so far!